src/select/select.js

```diff
@@ -289,27 +313,38 @@ class Select extends React.Component<PropsT, StatelessStateT> {
             }
           />
         )}
-        {selectedOptions.map(option => (
-          <Tag key={option.id} $multiple={multiple}>
-            {this.getSelectedOptionLabel(option)}
-            {multiple && (
-              <SearchIcon
-                onClick={e => {
+        {selectedOptions.map(option => {
+          const events = disabled
+            ? {
+                onClickCapture: e => e.stopPropagation(),
```

Comment:
What benefit does stopPropagation have for all of these cases?

It seems like there might be cases where you still want the vent to bubble up to the document. For example, if I have a popover that's visible, and it uses onClickOutside-style logic to close when the document is clicked. In theory clicking a disabled select should still cause probably still be able to trigger this behavior.

Reply:
do we try to do two actions via one click? I am not sure that click inside of Select should trigger close of popover somewhere else. Otherwise in general practice activating two event handlers at the very same moment may cause issues with controversy of application state if both change this state in the same data

Reply:
>I am not sure that click inside of Select should trigger close of popover somewhere else. 

This seems pretty common. For instance, in gmail click on the user dropdown to open that popover and then click on search - the popover will close. Twitter/facebook have similar behavior.

In these cases clicking anywhere outside the popover is essentially simulating a "blur" event for the popover, which doesn't natively support blur. By stopping propagation here you could be preventing that from happening.

Is there any downside to just not specifying a handler here when select is disabled? (instead of explicitly capturing the event and stopping propagation)

Reply:
Yes, if we don't specify handler, the click will sink down and activates logic of inner used component.
Unfortunately, we cannot stop propagation only downwards, and allow it bubble up on parent element. Event broadcast cycle stops it for all the elements and document is somewhere down the hole

Reply:
> The click will sink down and activates logic of inner used component.

That seems fine though? If the inner Input is `disabled` then it shouldn't have any interaction events. And if it does that means Input is broken and should be fixed.

Is there anything that fundamentally doesn't work correctly when you remove the stopPropagation? Seems like we can test this. If it works fine then maybe we should leave it out since it could break onClickOutside logic for other components.

Reply:
so about this particular case. Currently we don't send `disabled` to `<Tag>` so it means that tag will never deactivate it's cross button click itself. So there are two options:
- current solution assuming we don't trust Tag to handle `disabled` state
- pass `disabled` to Tag and assume it knows what to do, so we don't need to capture it ahead.

Reply:
It seems like we should pass disabled to Tag. That way it doesn't show a `pointer` cursor on the tag when the select element is disabled.